deploy/obc/storage_class.yaml

```
@@ -1,6 +1,8 @@
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
+  annotations:
+    description: Provides Object Bucket Claims(OBCs)
```

Perhaps better to separate the parenthesis from the last word?
```suggestion
    description: Provides Object Bucket Claims (OBCs)
```